src/wp-admin/includes/class-wp-comments-list-table.php

```diff
@@ -100,6 +100,7 @@ public function prepare_items() {
 		}
 
 		$comment_type = ! empty( $_REQUEST['comment_type'] ) ? $_REQUEST['comment_type'] : '';
+		$comment_type = 'note' === $comment_type ? '' : $comment_type;
```

Comment:
```suggestion
		$comment_type = '';
		
		if ( ! empty( $_REQUEST['comment_type'] ) && 'note' !== $_REQUEST['comment_type'] ) {
			$comment_type = $_REQUEST['comment_type'];
		}
```

This is a nit, but what about something like this to avoid two ternary operators for the same variable? Also avoids putting an `&&` condition in the one that's currently there.

Reply:
Works for me!